src/super_gradients/training/metrics/detection_metrics.py

```diff
@@ -179,7 +181,8 @@ def update(self, preds, target: torch.Tensor, device: str, inputs: torch.tensor,
         targets = target.clone()
         crowd_targets = torch.zeros(size=(0, 6), device=device) if crowd_targets is None else crowd_targets.clone()
 
-        preds = self.post_prediction_callback(preds, device=device)
```

Comment:
Leftover?

Reply:
Nope, just added the option to pass None as a post_prediction_callback in the case we want to pass the predictions as they are (which is what we need when we use the slidding window since NMS already happens inside).
I am addressing this in the docs in this PR on this page.